### PR TITLE
A Tiny JS World

### PR DESCRIPTION
**Pull request by iva-stasia**
### A Tiny JS World

[Demo](https://iva-stasia.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/iva-stasia/a-tiny-JS-world)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by iva-stasia** [comment]
Checked my code. It looks like there are no common mistakes.

**Comment by iva-stasia** [comment]
I realized that I can make my code shorter and got rid of the extra array.

File: submissions/iva-stasia/tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   saying: 'What\'s up?'
+};
+
+const catWomen = {
+   species: 'cat-human',
+   name: 'Dita',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: cat.saying
```
Comment (artem-trubin) [inline submissions/iva-stasia/tiny-js-world/index.js:45]: Your catwoman's saying should change when cat's saying changes. Your's does not. You only take the initial value of the `cat.saying`.

**Review by artem-trubin — CHANGES_REQUESTED** [review]
Nice work, you're almost there. Let's fix something, check the comment below.

File: submissions/iva-stasia/tiny-js-world/index.js (hunk 1 above)
Comment (iva-stasia) [reply submissions/iva-stasia/tiny-js-world/index.js]: Now I understand that it only works once when I create the object.
Changed it.

**Review by artem-trubin — APPROVED** [review]
Good job!

**Pull request merged**